morgoth/base/static/js/actions/addons.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
```

Comment:
I guess I could switch to just using fetch. I suspect the limitations of fetch won't be an issue for us.